packaging/obs/grafana-sap-hana-dashboards/grafana-sap-hana-dashboards.changes

```diff
@@ -1,4 +1,13 @@
 -------------------------------------------------------------------
+Fri Jun 11 19:30:21 UTC 2021 - Bernd Schubert <bschubert@suse.com>
+
+- Release 1.0.3
+  * The HANA system replication history panel could not show data, because of 
+    the node_name variable does not match to the output string from the exporter. 
+    Changing "host=" value to "src_host=" solves the issue. Fix is tested with 
+    HANA SPS040 and HANA SPS050.
+  * bgz#1187243
```

Comment:
Hey @pirat013 , sorry for being picky, but I would prefer to have the bugzilla reference within parenthesis.
`(bgz#1187243)` and with the same indentation as the text.

Example: https://github.com/SUSE/saphanabootstrap-formula/blob/master/saphanabootstrap-formula.changes#L41